### PR TITLE
Simple view of Sandbox state

### LINKED ISSUE #1120: Minimal, read-only visualisation of sandbox state
- [ ] Use TzKt
- [ ] Do not write Taqueria code to setup TzKt, just run it in terminal and connect it to the Sandbox
- [ ] Show implicit Accounts
  - [ ] Their operations
- [ ] Show contracts
  - [ ] Their entrypoints
  - [ ] Their operations
- [ ] We cannot invoke the entrypoints at the moment

### PR DESCRIPTION
**Pull request by AlirezaHaghshenas**
closes #1120

### CODE REVIEW

**Review by jchenche — APPROVED** [review]
LGTM!

File: taqueria-plugin-flextesa/package.json [hunk 1]
```diff
@@ -38,6 +38,7 @@
 	},
 	"dependencies": {
 		"@taqueria/node-sdk": "^0.12.0",
+		"@taqueria/protocol": "^0.12.0",
```
Comment (GImbrailo) [inline taqueria-plugin-flextesa/package.json:41]: Should these not be referencing `^0.14.0`?

File: taqueria-vscode-extension/package.json [hunk 2]
```diff
@@ -526,11 +574,15 @@
 		"vsce": "^2.6.4"
 	},
 	"dependencies": {
+		"@microsoft/signalr": "^6.0.8",
 		"@taqueria/protocol": "^0.12.0",
```
Comment (GImbrailo) [inline taqueria-vscode-extension/package.json:578]: Same as above

File: taqueria-plugin-flextesa/package.json (hunk 1 above)
Comment (AlirezaHaghshenas) [reply taqueria-plugin-flextesa/package.json]: Now I rebased on main and all are `0.14.4`

File: taqueria-vscode-extension/package.json (hunk 2 above)
Comment (AlirezaHaghshenas) [reply taqueria-vscode-extension/package.json]: Changed to `0.14.4`

**Review by GImbrailo — APPROVED** [review]
LGTM

**Pull request merged**